### DISCUSSION TITLE
Extension package.json "contributes", "keybindings" parsing appears to be broken when using "Win" key in sequences.

### DISCUSSION THREAD

**Discussion by ironsheep** [microsoft/vscode-discussions#1200] [Extension Development QnA] [answered]
Up for offering suggestions for key stroke mapping issues i’m see in package.json?

I’m trying to specify use of the Win key in my extension when used on Windows Systems.

I’m observing that if I used in “key”: “Ctrl+Win+n” for instance, it will work but only if “linux”: “…” is NOT present
likewise, “windows”: “Ctrl+Win+n” doesn’t work.  I’m trying to use the Win key because there are so many conflicts on Windows systems with VSCode built-in keystroke sequences.
Also, I’d like to file this as an issue/bug report, but I’ve no idea as to which repo/issues I should be filing it against.

To be more specific:
```
        "key": "Ctrl+Win+g",
        "mac": "Ctrl+Alt+g",  # this works but Linux shows as Ctrl+super+g
      "linux": "Ctrl+Alt+g",  # if i add this line then the windows key becomes Alt on windows?!
```

Now, to be more correct, I was thinking that this is what is actually needed:
```
        "key": "Ctrl+Alt+g",
    "windows": "Ctrl+Win+g"    # as this is really what I want for each platform, but this also generates an Alt key for Windows instead of the intended Windows key.  Likely a parser error?
```

So,  thoughts? And to which repo should this be reported? (edited)

**Comment by tamuratak** [comment, marked as answer]
> “windows”: “Ctrl+Win+n” doesn’t work

It must be `"win": “Ctrl+Win+n”`, not `"windows": ...`.

See 

- https://github.com/microsoft/vscode/blob/f80875f68108d4e82e72068e31aba6f29fbee735/src/vs/workbench/services/keybinding/browser/keybindingService.ts#L125
